### PR TITLE
v0.6.0 - Updating dependencies

### PR DESCRIPTION
**Pull request by theninj4**
I've updated all our dependencies and updated some of the test assertions to match new error messages introduced in later versions of jsonapi-server.

### CODE REVIEW

**Comment by theninj4** [comment]
Travis is having loads of issues with CORS - I'm not sure how they're running things, but the mocha tests and local karma tests pass and this is the SauceLabs perspective:

```
Chrome 51.0.2704 (Windows 10 0.0.0): Executed 28 of 29 (skipped 1) SUCCESS (32.865 secs / 28.347 secs)
Firefox 46.0.0 (Windows 10 0.0.0): Executed 28 of 29 (1 FAILED) (skipped 1) (39.995 secs / NaN secs)
Chrome Mobile 39.0.0 (Android 5.1.0): Executed 28 of 29 (1 FAILED) (skipped 1) (29.536 secs / NaN secs)
Mobile Safari 9.0.0 (iOS 9.2.0): Executed 21 of 29 (19 FAILED) (skipped 1) (20.629 secs / NaN secs)
Mobile Safari 7.0.0 (iOS 7.1.0): Executed 28 of 29 (skipped 1) SUCCESS (30.651 secs / 27.625 secs)
IE 11.0.0 (Windows 10 0.0.0): Executed 28 of 29 (skipped 1) SUCCESS (20.574 secs / 18.189 secs)
IE 9.0.0 (Windows 7 0.0.0): Executed 28 of 29 (skipped 1) SUCCESS (18.944 secs / 17.093 secs)
```

Safari 9.2 has always error'd out with CORS errors, the other 2x test fails look to be tests changing data and conflicting with one another (they currently run in parallel, not ideal but it does prove the JS code functions across these browsers).

File: test/_testServer.js [hunk 1]
```diff
@@ -3,6 +3,13 @@
 var jsonApiTestServer = require("jsonapi-server/example/server.js");
 if (!jsonApiTestServer.start) return;
 
+jsonApiTestServer.getExpressServer().use(function (req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Methods", "POST, GET, PUT, DELETE, OPTIONS");
+  res.header("Access-Control-Allow-Headers", "Content-Type");
+  next();
+});
```
Comment (theninj4) [inline test/_testServer.js:11]: Even with all these headers, iOS on SauceLabs throws CORS errors :'( As does Firefox on the Travis instance...

**Comment by pmcnr-hx** [comment]
👍 What do you think of adding Greenkeeper to the jsonapi projects?

**Comment by theninj4** [comment]
I like the idea, I'll look into it :+1:

**Pull request merged**